tests/sentry/integrations/slack/test_event_endpoint.py

```diff
@@ -130,61 +112,47 @@ def test_valid_event(self, check_signing_secret_mock):
 
 class LinkSharedEventTest(BaseEventTest):
     @responses.activate
-    def test_valid_token(self):
+    @patch(
+        "sentry.integrations.slack.event_endpoint.match_link",
+        # match_link will be called twice, for each our links. Resolve into
+        # two unique links and one duplicate.
+        side_effect=[
+            ("mock_link", {"arg1": "value1"}),
+            ("mock_link", {"arg1", "value2"}),
+            ("mock_link", {"arg1": "value1"}),
+        ],
+    )
+    @patch(
+        "sentry.integrations.slack.event_endpoint.link_handlers",
+        {
+            "mock_link": Handler(
+                matcher=re.compile(r"test"),
+                arg_mapper=make_type_coercer({}),
+                fn=Mock(return_value={"link1": "unfurl", "link2": "unfurl"}),
+            )
+        },
+    )
```

Comment:
This makes the test more of a unit test where we mock out the details of link handlers